<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1518</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QComboBox" name="AudioDevComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>音频设备:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AudioFilelNameEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>音频文件:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AudioFileBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>951</width>
     <height>1141</height>
    </rect>
   </property>
   <widget class="QPushButton" name="WakeTestBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WakeTestPauseBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WakeTestStopBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>测试项选择:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="TestModeComboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>唤醒测试</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>字准率测试</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="PlayModeComboBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>单曲播放</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>单曲循环</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>顺序播放</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>顺序循环</string>
     </property>
    </item>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>921</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="WakeIntervalTimeEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>唤醒次数:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="WakeDevCountEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="WakePlayCycleCountEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>间隔时间:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>唤醒率:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>播放次数:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="WakeupRateEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="GetWakeTestResultBtn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>260</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>获取唤醒统计次数</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>播放完成时间:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="WakePlayFinishTimeEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>唤醒完成时间:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="WakeFinishTimeEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>唤醒延迟时间(ms):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="WakeDelayTimeEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SetPlaySumBtn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>设置总播放次数</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>播放时间设置</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SetPlaySumEdit">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>60</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="WakePlayCycleCountEdit_3">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="CalcDelayTimeBtn">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>260</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>计算延迟时间</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ClearDevWakeRecordBtn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>110</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>清除设备唤醒记录</string>
      </property>
     </widget>
     <widget class="QLabel" name="UpdateTimeLabel">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>210</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;新宋体&quot;;
color: rgb(255, 0, 0);
font: 75 12pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="TimeCompensationEdit">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>310</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>310</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>补偿时间参数(ms):</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DisplayDateTimeBtn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>210</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>显示时间</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="CharTestPalyCountEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>120</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="CharTestIntervalTimeEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>播放次数:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>字准相似度:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>时间间隔:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GetCharTestDataBtn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>120</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>获取字准数据计算</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>931</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearTextEditBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清空日志</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveLogFileBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>保存日志到文件</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>541</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AudioFileBulkBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>批量导入</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>播放列表:</string>
   </property>
  </widget>
  <widget class="QSlider" name="VolumeSlider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>音量调节:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
